recipes/mpiFFT4py/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mpiFFT4py" %}
```

Comment:
FWIW, I am -1 on casing package names because of some platforms case insensitivity.